scripts/registry-census/build_matrix.py

```
@@ -0,0 +1,158 @@
+                n_files += 1
+                if ENTRY_RX.search(t):
+                    entries.append('./packs/' + safe + '/' + rel.replace(os.sep, '/'))
+        entries.sort()
+        truncated = len(entries) > 60
+        entries = entries[:60]
+        if not entries:
+            shutil.rmtree(dst, ignore_errors=True)
+            manifest[pack] = {'skipped': 'no extension-shaped JS'}
+            continue
+        spec = SPEC.format(pack=pack, safe=safe, entries=json.dumps(entries))
+        open(os.path.join(DEST, f'{safe}.matrix.test.ts'), 'w').write(spec)
```

Ruff (0.16.1)

[warning] Use a context manager for opening files

(SIM115)